health/REFERENCE.md

```diff
@@ -0,0 +1,732 @@
+| [`exec`](#alarm-line-exec)                          | no              | The script to execute when the alarm changes status.                                  |
+| [`delay`](#alarm-line-delay)                        | no              | Optional hysteresis settings to prevent floods of notifications.                      |
+| [`repeat`](#alarm-line-repeat)                      | no              | The interval for sending notifications when an alarm is in WARNING or CRITICAL mode.  |
+| [`option`](#alarm-line-option)                      | no              | Add an option to not clear alarms.                                                    |
+
+The `alarm` or `template` line must be the first line of any entity.
+
+#### Alarm line `alarm` or `template`
+
+This line starts an alarm or template based on the [entity type](#entity-types) you're interested in creating.
+
+**Alarm:**
+
+```yaml
+alarm: NAME
+```
+
+**Template:**
+
+```yaml
+template: NAME
+```
+
+`NAME` can be anything, with `.` (period) and `_` (underscore) as the only allowed symbols.
+
+#### Alarm line `on`
+
+This line defines the chart this alarm should attach to.
+
+**Alarms:**
+
+```yaml
+on: CHART
+```
+
+The value `CHART` should be the unique ID or name of the chart you're interested in, as shown on the dashboard. In the
+image below, the unique ID is `system.cpu`.
+
+![Finding the unique ID of a
+chart](https://user-images.githubusercontent.com/1153921/67443082-43b16e80-f5b8-11e9-8d33-d6ee052c6678.png)
+
+**Template:**
+
+```yaml
+on: CONTEXT
+```
+
+The value `CONTEXT` should be the context you want this template to attach to.
+
+Need to find the context? Hover over the date on any given chart and look at the tooltip. In the image below, which
+shows a disk I/O chart, the tooltip reads: `proc:/proc/diskstats, disk.io`.
+
+![Finding the context of a chart via the tooltip](https://user-images.githubusercontent.com/1153921/68882856-2b230880-06cd-11ea-923b-b28c4632d479.png)
+
+You're interested in what comes after the comma: `disk.io`. That's the name of the chart's context. 
+
+If you create a template using the `disk.io` context, it will apply an alarm to every disk available on your system.
+
+#### Alarm line `os`
+
+The alarm or template will be used only if the operating system of the host matches this list specified in `os`.
+
+The value is a space-separate list of simple patterns. You can use `*` as a wildcard or prefix with `!` for a negative
+match. Order is important, too!
+
+See our [simple patterns docs](../libnetdata/simple_pattern/) for more examples.
+
+The following example enables the entity on Linux, FreeBSD, and MacOS, but no other operating systems.
+
+```yaml
+os: linux freebsd macos
+```
+
+#### Alarm line `hosts`
+
+The alarm or template will be used only if the hostname of the host matches this list.
+
+The value is a space-separate list of simple patterns. You can use `*` as a wildcard or prefix with `!` for a negative
```

Comment:
The line 141 looks like a copy and paste from line 126. Is it not better to do the defintion in the introduction and say something like "this value is space-separate list"?

Reply:
Good idea. I added a description of space-separated lists higher up in the document, along with a link to the simple patterns doc, and just said "this value is a space-separated list" when describing each line.